docs/api/non_geo_datasets.csv

```diff
@@ -37,6 +37,8 @@ Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
 `SpaceNet`_,I,WorldView-2/3 Planet Lab Dove,"1,889--28,728",2,102--900,0.5--4,MSI
 `SSL4EO`_-L,T,Landsat,1M,-,264x264,30,MSI
 `SSL4EO`_-S12,T,Sentinel-1/2,1M,-,264x264,10,"SAR, MSI"
+`SSL4EOLBenchmark`_-TM-TOA & CDL, S, Lansat, 25k,18,264x264,30,MSI
```

Comment:
@adamjstewart If the row entry looks okay to you, I will add the other benchmark dataset combinations

Reply:
Let's just add a single row for "Landsat & NLCD & CDL"

Reply:
maybe one for NLCD and one for CLD because they have different number of classes?

Reply:
Sure